views/portfolio.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset="utf-8">
+    <link href="../styles/normalize.css">
+    <link href="https://fonts.googleapis.com/css?family=Space+Mono" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Work+Sans" rel="stylesheet">
+    <link href="../styles/main.css" rel="stylesheet">
+    <title>guillermina</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+
+        </nav>
+    </header>
+    <section class="header_box">
+
+    </section>
+    <section class="main_box">
+        <h2>hi i'm guillermina muro,</h2>
```

Comment:
you can probably make this an `<h1>` instead of an h2 and style it the way you want to. h1 is for the most important information on the page, which this is